### PR TITLE
Get version info from ClusterVersion

### PR DESCRIPTION
**Pull request by liangxia**
We have to specify version info when trigger automation. We would like to get the version info automatically (from ClusterVersion).

### CODE REVIEW

**Comment by pruan-rht** [comment]
@akostadinov PTAL

File: lib/environment.rb [hunk 1]
```diff
@@ -278,11 +278,14 @@ def parse_version(ver_str)
...
-        return opts[:version], @major_version, @minor_version
+        _version = opts[:version]
+      else
+        _version = admin.cli_exec(:get, resource: "clusterversion", resource_name: "version", o: "jsonpath={.status.desired.version}")
       end
+      @major_version, @minor_version = parse_version(_version)
+      return _version, @major_version, @minor_version
 
-      raise "getting version of cluster as user not possible presently"
+      #raise "getting version of cluster as user not possible presently"
```
Comment (akostadinov) [inline lib/environment.rb:288]: We already set default version in `config/config.yaml` so we need to comment out that value if we want this to work OOB.

Also I think we better do like this:
```
if opst[:version]
  ...
elsif admin?
  ...
else
  raise "cluster version not set and getting without admin access not possible presently"
end
```

Comment (liangxia) [reply lib/environment.rb]: @akostadinov  Updated.

**Comment by akostadinov** [comment]
Thank you!
/lgtm
/unhold

**Pull request merged**